dns/dnscrypt-proxy/src/opnsense/mvc/app/controllers/OPNsense/Dnscryptproxy/forms/general.xml

```diff
@@ -110,7 +110,7 @@
         <help>Disable TLS session tickets - increases privacy but also latency.</help>
     </field>
     <field>
-        <id>general.fallback_resolver</id>
+        <id>general.bootstrap_resolvers</id>
```

Comment:
can't just rename this ;)

Reply:
Why? Don't you want the UI config and its configuration to be consistent?

Reply:
Well, you are changing existing setting's storage locations. These can be renamed but require work to be set from the old values. As it was in the original PR the setting is not retained. We can talk about adjusting labels in the scope of this PR if it helps though.

Reply:
Is there a guide somewhere on how to migrate old values?